src/config/index.ts

```diff
@@ -1,6 +1,5 @@
 import { mergeAll } from "lodash/fp";
-import * as Types from "./types";
-export * from "./types";
+import * as Types from "@types/index";
```

Comment:
what is wrong here? 

It says

```
[tsl] ERROR in /home/uer/f/code/dw-kid4/src/config/index.ts(2,24)
      TS6137: Cannot import type declaration files. Consider importing 'index' instead of '@types/index'.
```